Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -182,11 +182,11 @@ private static GoogleCredential CreateDefaultCredentialFromJson(JsonCredentialPa
             {
                 case JsonCredentialParameters.AuthorizedUserCredentialType:
                     return new GoogleCredential(CreateUserCredentialFromJson(credentialParameters));
-
+#if !NETSTANDARD
                 case JsonCredentialParameters.ServiceAccountCredentialType:
                     return GoogleCredential.FromCredential(
```

Comment:
Should we throw an exception here with a suitable error message instead? Something like _"Service Account credentials are not supported in .NET Core."_ or something like that?